Policies/github-terms-of-service.md

```diff
@@ -136,25 +159,33 @@ Some Accounts may have private repositories, which allow the User to control acc
 GitHub considers the contents of private repositories to be confidential to you. GitHub will protect the contents of private repositories from unauthorized use, access, or disclosure in the same manner that we would use to protect our own confidential information of a similar nature and in no event with less than a reasonable degree of care.
 
 #### 3. Access
-GitHub employees may only access the content of your private repositories in the following situations:
+GitHub employees may only access the content of your private repositories in the following situations (if you don't care what those situations are, you can skip reading the following bullet points):
```

Comment:
:-1:

Reply:
My comment https://github.com/github/site-policy/pull/204#discussion_r355114181 also applies here. If that comment doesn't sufficiently address @KOLANICH's comment, please then can @KOLANICH elaborate on exactly what is the point of contention here?

Reply:
The same: I can ignore any part of ToS I want to ignore. It is obvious and the placing this obvious fact into the ToS just makes it larger.